index.html

```diff
@@ -937,9 +947,58 @@ <h3><em>Detecting &amp; matching direction (TBD)</em></h3>
 
 <p class="reviewComments"><a href="https://github.com/w3c/i18n-activity/labels/t%3Abidi_detection_x" target="_blank">See related review comments.</a></p>
 </section>
-</section>
 
+<section id="strcat" class="subtopic">
+<h3>Concatenation of strings</h3>
 
+<p class="reviewComments"><a href="https://github.com/w3c/i18n-activity/labels/t%3Astrcat" target="_blank">See related review comments.</a></p>
+
+<div class="req" id="strcat_anti_pattern">
+	<p class="advisement">Specifications SHOULD NOT require the concatenation of string values to form [=natural language=] or displayable string values.</p>
+</div>
+
+<p>Creation of [=natural language=] text values by concatenating multiple strings together is an internationalization anti-pattern. Languages vary greatly in word order, count, grammatical gender or case, punctuation, and many other requirements. As a result, avoid requiring or suggesting that implementations generate human-readable messages from sub-strings.</p>
+
+<div class="xref"><span class="seealso">See also</span>
+   <p><cite><a href="https://www.w3.org/International/articles/composite-messages/index.en.html">Working with Composite Messages</a></cite></p>
+</div>
+
+<div class="req" id="bidi_gen_advice">
+	<p class="advisement">Specifications SHOULD provide guidance to implementers on avoiding problems with text directionality for any field(s) that the specification requires an implementation to create or generate for display to users.</p>
+</div>
+
+<p>Specifications for APIs, protocols, or document formats sometimes require an implementation to create or provide a field containing a display name or description. When such a string is assembled from separate parts, it can result in problems with presentation or understanding due to the way that the <cite>Unicode Bidirectional Algorithm</cite> [[UAX9]] processes the assembled string. In such cases, the specification should guide implementers about how to create values that will display properly.</p>
```

Comment:
Several things are unclear to me here, and rather than provide text suggestions that may be barking up the wrong tree, we should perhaps discuss during our telecon.  Questions include: 
- do you mean bidrectional text or text that generally runs in a particular direction (eg. RTL)?
- i suspect, but i'm not sure, that we are talking solely about strings that contain bidirectional text within them, or about correctly displaying any string that is inserted into a target location by the consumer?
- if the latter, we're focusing on what producers need to do, we should mention dirname or using other metadata

Basically, i think we need a little more clarity about the role of producers vs consumers, and in-string direction vs how a string will be inserted into a target location.  It seems a little vague at the moment.  The section title, Concatenation of Strings, doesn't make it sound like we're concerned with string internal direction, but rather strings being inserted into the target location.  The example shows one scenario that fits the section title, but the preceding text needs to be clearer before we get to the example, and needs to clarify whether we are also talking about string-internals.

Reply:
added to the agenda for 2023-03-30